src/exercise/04.html

```diff
@@ -1,22 +1,30 @@
-<!-- Creating custom components -->
```

Comment:
No harm removing this as it'll be caught by the ending comment further down (as a multiline comment) though it's unusual to have a `<!--` on a subsequent line on a multiline comment